torchao/quantization/quant_primitives.py

```diff
@@ -790,16 +1058,152 @@ def choose_qparams_affine(
         eps,
         scale_dtype,
         zero_point_dtype,
-        preserve_zero,
-        zero_point_domain.name,
     )
 
 
+@torch.no_grad()
+def choose_qparams_affine_tinygemm(
+    input: torch.Tensor,
+    mapping_type: MappingType,
+    block_size: Tuple[int],
+    target_dtype: torch.dtype,
+    quant_min: Optional[Union[int, float]] = None,
+    quant_max: Optional[Union[int, float]] = None,
+    eps: Optional[float] = None,
+    scale_dtype: Optional[torch.dtype] = None,
+    zero_point_dtype: Optional[torch.dtype] = None,
+) -> Tuple[torch.Tensor, torch.Tensor]:
+    """
+    Specialized version of choose_qparams_affine with zero_point_domain=ZeroPointDomain.FLOAT and preserve_zero=False.
```

Comment:
I think we probably don't want to mention zero_point_domain and preserve_zero here, but explain the meaning